### ISSUE TITLE
[Suggestion] Filter for last X matches

### ISSUE THREAD

**Issue by Iorce** [bippum/listen-bot#51]
Example:
--matches as \<Hero\> last 5

Lists the last 5 matches as \<Hero\>

--hero as \<Hero\> last 5

Lists hero stats for the last 5 matches as \<Hero\>

**Comment by bippum** [comment]
I'm going to need you to elaborate more on this one. I see something here, I just don't quite understand exactly what you would like to see.

**Comment by Iorce** [comment]
Forgot to escape \<\>.

I came up with that idea when I was in a discussion about Naga Siren (which I usually don't play) and in fact have such a low amount of games that half of my games are from 5 years ago.

While date filtering would be the obvious choice now that I think of it again, I thought about displaying only the last 7 or so matches that were in the current year to have a more accurate representation of me as that hero at this time.

So in short:

- Display the last X matches played as a given hero

- Display hero stats for the last X matches as given hero

(- Display overall stats for the last X matches)

**Comment by bippum** [comment]
Finished in bbe98808b1df568dcd10716fbb504b947fa6afa8 I think.
